google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -31,6 +31,18 @@
     pol.member_domain, pol.member_name
 """
 
+ORGANIZATIONS = """SELECT org_id, name, display_name, lifecycle_state,
```

Comment:
nit: do you think it's more readable if the select line is on it's own line? i.e. the statements are more lined up.